tests/beaker_tests/cisco_command_config/test_command_config.rb

```diff
@@ -193,7 +193,7 @@ def test_set_get
   logger.info('* check config')
   on(agent, cmd_prefix + "test_get='incl loopback1'")
   fail_test("TestStep :: set/get :: FAIL\nstdout:\n#{stdout}") unless
-    stdout[/^interface loopback1/]
+    stdout[/^?\n?interface loopback1/]
```

Comment:
Can you add some sample output from both agent versions that this is trying to match?  Also add comments to make it clear that we need to match two versions of output.